### PR TITLE
added potplayer

### PR DESCRIPTION
**Pull request by Dalet**
there doesn't seem to be args for on top and subtitles

### CODE REVIEW

**Comment by mafintosh** [comment]
couldn't we check if the 64 bit version was installed and use that instead of having two options?

**Comment by Dalet** [comment]
You can have both versions installed though, and they're separate downloads on https://potplayer.daum.net/. I have both installed for instance. What do you think?

**Comment by mafintosh** [comment]
If you have both is there any reason to not just always use the 64 bit one?

**Comment by mafintosh** [comment]
I wanna keep the options as simple as possible in general :)

**Comment by Dalet** [comment]
Will do once I can install peerflix again https://github.com/mafintosh/peerflix/issues/184
If you want to do it, it needs to check if `HKCU/SOFTWARE/DAUM/PotPlayer64` exists.

**Comment by Dalet** [comment]
There we go, and I found how to add subtitles in command line. :smile:

File: app.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ var argv = rc('peerflix', {}, optimist
   .alias('m', 'mplayer').describe('m', 'autoplay in mplayer*').boolean('m')
   .alias('g', 'smplayer').describe('g', 'autoplay in smplayer*').boolean('g')
   .describe('mpchc', 'autoplay in MPC-HC player*').boolean('boolean')
+  .describe('potplayer', 'autoplay in MPC-HC player*').boolean('boolean')
```
Comment (mafintosh) [inline app.js]: this should say 'autoplay in potplayer' right?

**Comment by Dalet** [comment]
Should be good now?

**Comment by mafintosh** [comment]
:+1:

**Comment by mafintosh** [comment]
Released in 0.27.0

**Comment by mafintosh** [comment]
whoops. i forgot to merge before releasing. out in 0.28.0

**Pull request merged**